goldmane/fv/daemon_test.go

```diff
@@ -290,3 +290,72 @@ func TestHints(t *testing.T) {
 		return nil
 	}, 5*time.Second, 1*time.Second).Should(Succeed())
 }
+
+func TestStatistics(t *testing.T) {
+	cfg := daemon.Config{
+		LogLevel:                 "debug",
+		Port:                     8988,
+		AggregationWindow:        time.Second * 1,
+		EmitAfterSeconds:         2,
+		EmitterAggregationWindow: time.Second * 2,
+	}
+	defer daemonSetup(t, cfg)()
```

Comment:
Looks like `deamonSetup` function initialize global `ctx` field. Might be better to make `deamonSetup` return `ctx` and use local `ctx` variable. It will siplyfy runnig FVs in parallel, if we need it.

Reply:
Yeah, it might be possible to run these in parallel with a bit of work - we'd need to commonize a lot more than the context though (for example, we'd need separate TCP ports for each test to bind to and logic to manage allocation of ports for each test).

This is relying on existing test infrastructure and refactoring that is I think outside the scope of this PR. I'm not against us taking time to investigate how difficult it would be to make these tests run in parallel as a separate task, though.